mini-e2e-helm.groovy

```diff
@@ -55,19 +55,19 @@ node('cico-workspace') {
 			timeout(time: 30, unit: 'MINUTES') {
 				env.NAMESPACE = 'cephcsi-e2e-' + UUID.randomUUID().toString().split('-')[-1]
 				ssh 'cd /opt/build/go/src/github.com/ceph/ceph-csi && ./scripts/install-helm.sh up'
-				ssh 'cd /opt/build/go/src/github.com/ceph/ceph-csi && ./scripts/install-helm.sh install-cephcsi ${NAMESPACE}'
+				ssh 'cd /opt/build/go/src/github.com/ceph/ceph-csi && ./scripts/install-helm.sh install-cephcsi ${env.NAMESPACE}'
```

Comment:
the `env.NAMESPACE` should set `NAMESPACE` in the environment that gets replaced when doing the `sh` command, using `env.NAMESPACE` should not be needed.

It may be better to fo a `def namespace = ...` at the top of the file, and use `{namespace}` where needed. Note that Groovy does not replace variables when using strings with `'blabla {namespace} ...'` single quotes, use `"` for that to work.